src/Wyam.Modules.Minification/CssMinify.cs

```diff
@@ -0,0 +1,59 @@
+﻿using System.Collections.Generic;
+using WebMarkupMin.Core;
+using Wyam.Common.Documents;
+using Wyam.Common.Modules;
+using Wyam.Common.Pipelines;
+
+namespace Wyam.Modules.Minification
+{
+    /// <summary>
+    /// Minifies the CSS content.
+    /// </summary>
+    /// <remarks>
+    /// This module takes the CSS content and uses minification to reduce the output.
+    /// </remarks>
+    /// <example>
+    /// <code>
+    /// Pipelines.Add("CSS",
+    ///     ReadFiles("*.css"),
+    ///     CssMinify(),
+    ///     WriteFiles(".css")
+    /// );
+    /// </code>
+    /// </example>
+    /// <category>Minification</category>
```

Comment:
Let's change all the categories to "Content" since that's what these modules transform (I'm trying to keep the number of categories low so we don't cause confusion).